biojava-structure/src/main/java/org/biojava/nbio/structure/io/cif/CifStructureConsumerImpl.java

```diff
@@ -831,6 +843,10 @@ public void consumePdbxStructOperList(PdbxStructOperList pdbxStructOperList) {
     public void consumeRefine(Refine refine) {
         for (int rowIndex = 0; rowIndex < refine.getRowCount(); rowIndex++) {
             // RESOLUTION
+        	ValueKind valueKind = refine.getLsDResHigh().getValueKind(rowIndex);
+        	if (ValueKind.NOT_PRESENT.equals(valueKind)) {
```

Comment:
`if (!ValueKind.PRESENT.equals(valueKind)) {` might be slightly safer here. Possible `ValueKind` values are `PRESENT`, `NOT_PRESENT`, and `UNKNOWN`. The latter 2 would resolve to 0.0 for floats.

Reply:
Thanks. Done.